app/models/template.rb

```diff
@@ -304,6 +304,10 @@ def template_changes
         !changes.include?('locked')
       errors.add(:base, _("This template is locked. Please clone it to a new template to customize."))
     end
+
+    if !modify_locked && locked? && !audit_comment.empty?
```

Comment:
So if the comment is present, you allow overwriting it? Is this expected behavior?

Reply:
It is not a regular attribute, it comes from Audited gem. It creates a new audit with a comment on each save.